gunicorn/config.py

```diff
@@ -396,6 +396,9 @@ def validate_class(val):
     return validate_string(val)
 
 
+getargspec = getattr(inspect, 'getfullargspec', inspect.getargspec)
```

Comment:
Sorry, I forgot to say this in my earlier comment, but could you please move this to `gunicorn/_compat.py`?